# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/speakersafetyd/hotfix-install-t.patch
# Copyright (C) 2023 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- speakersafetyd-1.1.2/Makefile.orig	2025-03-29 12:07:32.000000000 +0100
+++ speakersafetyd-1.1.2/Makefile	2025-06-22 20:57:43.382887656 +0200
@@ -23,7 +23,7 @@
 	install -dDm0755 $(DESTDIR)/$(UDEVDIR)
 	install -pm0644 95-speakersafetyd.rules $(DESTDIR)/$(UDEVDIR)/95-speakersafetyd.rules
 	install -dDm0755 $(DESTDIR)/$(SHAREDIR)/speakersafetyd/apple
-	install -pm0644 -t $(DESTDIR)/$(SHAREDIR)/speakersafetyd/apple $(wildcard conf/apple/*)
+	install -pm0644 $(wildcard conf/apple/*) $(DESTDIR)/$(SHAREDIR)/speakersafetyd/apple
 	install -dDm0755 -o $(SPEAKERSAFETYD_USER) -g $(SPEAKERSAFETYD_GROUP) $(DESTDIR)/$(VARDIR)/lib/speakersafetyd
 	install -dDm0700 -o $(SPEAKERSAFETYD_USER) -g $(SPEAKERSAFETYD_GROUP) $(DESTDIR)/$(VARDIR)/lib/speakersafetyd/blackbox
 	install -dDm0755 $(DESTDIR)/$(TMPFILESDIR)
